.github/workflows/config_coverage.yml

```diff
@@ -0,0 +1,150 @@
+          curl -sL http://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          sudo add-apt-repository -y "deb http://apt.llvm.org/$DISTRO_FANCYNAME/ llvm-toolchain-$DISTRO_FANCYNAME main"
+          echo "::endgroup::"
+
+          # Get the largest Clang package number available.
+          export LLVM_VER="$(apt-cache search --full 'clang-[[:digit:]]*$' | grep '^Package: clang' | cut -d ' ' -f 2 | sort -V | tail -n 1 | sed 's/clang-//')"
+          echo "::group::Install Clang and Clang-Tidy version ${LLVM_VER}"
+          sudo apt-get -y --no-install-recommends install \
+            clang-$LLVM_VER      \
+            clang-tidy-$LLVM_VER
+          sudo update-alternatives --install                   \
+            /usr/bin/clang clang /usr/bin/clang-$LLVM_VER 1000 \
+            --slave                                            \
+              /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-$LLVM_VER
+          echo "::endgroup::"
+
+          echo "Installed Clang:"
+          update-alternatives --query clang
+      - name: "Package CodeChecker"
+        id: codechecker
+        run: |
+          pushd analyzer
+
+          echo "::group::venv"
+          make venv
+          source venv/bin/activate
+          echo "::endgroup::"
+
+          echo "::group::CodeChecker package"
+          make standalone_package
+          deactivate
+          echo "::endgroup::"
+
+          echo "::set-output name=CODECHECKER_PATH::$(readlink -f ./build/CodeChecker/bin)"
+          popd # analyzer
+      - name: "Dump checker list"
+        run: |
+          export PATH="${{ steps.codechecker.outputs.CODECHECKER_PATH }}:$PATH"
+          CodeChecker analyzers --details
+          CodeChecker checkers \
+            --analyzer clangsa clang-tidy \
+            --output rows \
+              > checker_list_normal.txt
+          CodeChecker checkers \
+            --analyzer clangsa clang-tidy \
+            --warnings \
+            --output rows \
+              > checker_list_diagnostics.txt
+      - name: "Perform checker config coverage check (--warnings)"
+        continue-on-error: true
+        run: |
+          set +e # Do not hard exit on an erroring call!
+          .github/workflows/config_label_check.py \
+            "checker_list_diagnostics.txt" \
+            "config/labels/analyzers/clangsa.json" \
+            "config/labels/analyzers/clang-tidy.json" \
+            --existing-filter "clang-diagnostic-" \
+            --new-filter "clang-diagnostic-"
+          EXIT_STATUS=$?
+          echo "Coverage check returned: $EXIT_STATUS."
+          # Explicitly check if the bit for "8" is set in the result,
+          # indicating new checkers without severity set.
+          if [[ $(($EXIT_STATUS & 8)) -eq 8 ]]
+          then
+            echo "::warning title=New unconfigured diagnostics::The checker label config files lack some new diagnostic report (\"warning\") kinds."
+            exit 0
```

Comment:
How does this GitHub action indicate when some modification in our "checker label file" is needed?